ecommerce_integrations/controllers/customer.py

```diff
@@ -48,6 +48,13 @@ def sync_customer(self, customer_name: str, customer_group: str) -> None:
 		except Exception as e:
 			raise e
 
+	def get_customer_address_doc(self,customer_name: str, type):
```

Comment:
`type` is reserved keyword. Refrain from using it as variable name.

Reply:
cool, you see I am not familiar with python but it worked^^